crates/chain/src/tx_graph.rs

```diff
@@ -765,6 +791,14 @@ impl<A: Anchor> TxGraph<A> {
                 changeset.merge(self.insert_seen_at(txid, seen_at));
             }
         }
+        for txid in update.missing {
```

Comment:
I think it'd help to update `apply_update_at` documentation in order to mention the `evicted_at` too.